$OpenBSD: patch-configure,v 1.5 2012/05/16 19:35:22 landry Exp $
ensure vala and gjs are not picked as hidden deps
--- configure.orig	Thu Nov 19 23:20:25 2009
+++ configure	Wed May 16 21:33:00 2012
@@ -13036,81 +13036,6 @@ fi
 
 enable_vala=no
 
-pkg_failed=no
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for VALA" >&5
-$as_echo_n "checking for VALA... " >&6; }
-
-if test -n "$PKG_CONFIG"; then
-    if test -n "$VALA_CFLAGS"; then
-        pkg_cv_VALA_CFLAGS="$VALA_CFLAGS"
-    else
-        if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"vala-1.0\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "vala-1.0") 2>&5
-  ac_status=$?
-  $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
-  test $ac_status = 0; }; then
-  pkg_cv_VALA_CFLAGS=`$PKG_CONFIG --cflags "vala-1.0" 2>/dev/null`
-else
-  pkg_failed=yes
-fi
-    fi
-else
-	pkg_failed=untried
-fi
-if test -n "$PKG_CONFIG"; then
-    if test -n "$VALA_LIBS"; then
-        pkg_cv_VALA_LIBS="$VALA_LIBS"
-    else
-        if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"vala-1.0\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "vala-1.0") 2>&5
-  ac_status=$?
-  $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
-  test $ac_status = 0; }; then
-  pkg_cv_VALA_LIBS=`$PKG_CONFIG --libs "vala-1.0" 2>/dev/null`
-else
-  pkg_failed=yes
-fi
-    fi
-else
-	pkg_failed=untried
-fi
-
-
-
-if test $pkg_failed = yes; then
-
-if $PKG_CONFIG --atleast-pkgconfig-version 0.20; then
-        _pkg_short_errors_supported=yes
-else
-        _pkg_short_errors_supported=no
-fi
-        if test $_pkg_short_errors_supported = yes; then
-	        VALA_PKG_ERRORS=`$PKG_CONFIG --short-errors --errors-to-stdout --print-errors "vala-1.0"`
-        else
-	        VALA_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "vala-1.0"`
-        fi
-	# Put the nasty error message in config.log where it belongs
-	echo "$VALA_PKG_ERRORS" >&5
-
-	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-                enable_vala=no
-elif test $pkg_failed = untried; then
-	enable_vala=no
-else
-	VALA_CFLAGS=$pkg_cv_VALA_CFLAGS
-	VALA_LIBS=$pkg_cv_VALA_LIBS
-        { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-$as_echo "yes" >&6; }
-	enable_vala=yes
-fi
-if test "x$enable_vala" = "xyes"; then
-	VAPI_DIR=`pkg-config --variable=vapidir vala-1.0`
-else
-	VAPI_DIR=
-fi
  if test "x$enable_vala" = "xyes"; then
   ENABLE_VALA_TRUE=
   ENABLE_VALA_FALSE='#'
@@ -13682,76 +13607,7 @@ fi
 
 
 
-pkg_failed=no
-{ $as_echo "$as_me:${as_lineno-$LINENO}: checking for GJS" >&5
-$as_echo_n "checking for GJS... " >&6; }
-
-if test -n "$PKG_CONFIG"; then
-    if test -n "$GJS_CFLAGS"; then
-        pkg_cv_GJS_CFLAGS="$GJS_CFLAGS"
-    else
-        if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"gjs-1.0 >= 0.2\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "gjs-1.0 >= 0.2") 2>&5
-  ac_status=$?
-  $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
-  test $ac_status = 0; }; then
-  pkg_cv_GJS_CFLAGS=`$PKG_CONFIG --cflags "gjs-1.0 >= 0.2" 2>/dev/null`
-else
-  pkg_failed=yes
-fi
-    fi
-else
-	pkg_failed=untried
-fi
-if test -n "$PKG_CONFIG"; then
-    if test -n "$GJS_LIBS"; then
-        pkg_cv_GJS_LIBS="$GJS_LIBS"
-    else
-        if test -n "$PKG_CONFIG" && \
-    { { $as_echo "$as_me:${as_lineno-$LINENO}: \$PKG_CONFIG --exists --print-errors \"gjs-1.0 >= 0.2\""; } >&5
-  ($PKG_CONFIG --exists --print-errors "gjs-1.0 >= 0.2") 2>&5
-  ac_status=$?
-  $as_echo "$as_me:${as_lineno-$LINENO}: \$? = $ac_status" >&5
-  test $ac_status = 0; }; then
-  pkg_cv_GJS_LIBS=`$PKG_CONFIG --libs "gjs-1.0 >= 0.2" 2>/dev/null`
-else
-  pkg_failed=yes
-fi
-    fi
-else
-	pkg_failed=untried
-fi
-
-
-
-if test $pkg_failed = yes; then
-
-if $PKG_CONFIG --atleast-pkgconfig-version 0.20; then
-        _pkg_short_errors_supported=yes
-else
-        _pkg_short_errors_supported=no
-fi
-        if test $_pkg_short_errors_supported = yes; then
-	        GJS_PKG_ERRORS=`$PKG_CONFIG --short-errors --errors-to-stdout --print-errors "gjs-1.0 >= 0.2"`
-        else
-	        GJS_PKG_ERRORS=`$PKG_CONFIG --errors-to-stdout --print-errors "gjs-1.0 >= 0.2"`
-        fi
-	# Put the nasty error message in config.log where it belongs
-	echo "$GJS_PKG_ERRORS" >&5
-
-	{ $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-                enable_js=no
-elif test $pkg_failed = untried; then
 	enable_js=no
-else
-	GJS_CFLAGS=$pkg_cv_GJS_CFLAGS
-	GJS_LIBS=$pkg_cv_GJS_LIBS
-        { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-$as_echo "yes" >&6; }
-	enable_js=yes
-fi
  if test "x$enable_js" = "xyes"; then
   ENABLE_JS_TRUE=
   ENABLE_JS_FALSE='#'
